python/paddle/fluid/contrib/slim/tests/test_quantization_pass.py

```diff
@@ -138,29 +138,29 @@ def linear_fc_quant(self, quant_type):
             place=place,
             activation_quantize_type=quant_type)
         transform_pass.apply(graph)
-        marked_nodes = set()
-        for op in graph.all_op_nodes():
-            if op.name().find('quantize') > -1:
-                marked_nodes.add(op)
-        graph.draw('.', 'quantize_fc_' + quant_type, marked_nodes)
+        if not enable_ce:
```

Comment:
will this enable in windows CI, it looks like only disabled in ce?

Reply:
Yes, maybe the name is confusing. I will change it.